Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -248,6 +248,15 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output.shape, expected)
     }
 
+
+    func testFunction() {
+        let tanhLayer = Function<Tensor<Float>, Tensor<Float>>(tanh)
+        let input = Tensor(shape:[5, 1], scalars:(0..<5).map(Float.init))
```

Comment:
Add a space after each colon.
```suggestion
        let input = Tensor(shape: [5, 1], scalars: (0..<5).map(Float.init))
```